server/views/pages/communityOffenderManager/residentialChecks/tasklist.njk

```diff
@@ -27,7 +27,11 @@
                         title: {
                             text: task.taskName
                         },
-                        href: "#",
+                        href: paths.probation.assessment.curfewAddress.addressCheckTask | toPath({
```

Comment:
mmh.. guess this shouldn't be hardcoded for all tasks and should be in the meta data?